rules/Php81/Rector/Array_/FirstClassCallableRector.php

```diff
@@ -83,14 +90,58 @@ public function name()
      */
     public function getNodeTypes(): array
     {
-        return [Property::class, ClassConst::class, Array_::class];
+        return [Property::class, ClassConst::class, Array_::class, FuncCall::class];
     }
 
     /**
-     * @param Property|ClassConst|Array_ $node
+     * @param Property|ClassConst|Array_|FuncCall $node
      */
-    public function refactorWithScope(Node $node, Scope $scope): int|null|StaticCall|MethodCall
+    public function refactorWithScope(Node $node, Scope $scope): int|null|StaticCall|MethodCall|FuncCall
     {
+        if ($node instanceof FuncCall) {
+            if (! $node->name instanceof Name) {
+                return null;
+            }
+
+            if ($node->isFirstClassCallable()) {
+                return null;
+            }
+
+            $functionName = $node->name->toString();
+
+            try {
+                $rf = new ReflectionFunction($functionName);
+            } catch (ReflectionException) {
+                return $node;
+            }
+
+            $callableArgs = [];
+
+            foreach ($rf->getParameters() as $rp) {
+                if ($rp->getType() instanceof ReflectionNamedType && $rp->getType()->getName() === 'callable') {
+                    $callableArgs[] = $rp->getPosition();
+                }
+            }
+
+            /** @var Arg $arg */
+            foreach ($node->getRawArgs() as $key => $arg) {
```

Comment:
It can use `$node->getArgs()` so it always `Arg`, so `@var Arg $arg` is not needed